packages/design-tokens/index.stories.tsx

```diff
@@ -1,56 +1,452 @@
-import { Box } from "@kaizen/component-library"
-import { Divider } from "@kaizen/draft-divider"
-import { Meta } from "@storybook/react"
+/* Stories Modules */
```

Comment:
This is 100% not the cleanest code in the world, but didn't want to spend too much time on it.
We can either merge this and come back to it, or just not merge it yet.